.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
```

Comment:
I'm not sure what you mean.

> `root = true`
This line tells EditorConfig this is the root of the project, thus EditorConfig won’t apply the settings of .editorconfig outside this directory.

Look like it's ok.